### DISCUSSION TITLE
Shorter form for command initiators in toolkit

### DISCUSSION THREAD

**Discussion by samhere17** [wrappid/.github#3] [Q&A] [answered]
In our toolkit, typing `wrappid` and `wrappid-dev` takes time. I suggested to have smaller versions for both.

- For `wrappid` : I suggested `wcli` & `wc`
- For `wrappid-dev` : I suggested `wdev` & `wd`

Refer https://github.com/wrappid/toolkit/issues/8

`wcli`, `wdev` & `wd` has been implemented. `wc` happens to be a internal linux command.

Please suggest alternatives for `wc`

**Comment by samhere17** [comment]
@wrappid/testers please suggest

**Reply by saifalamkhan373** [reply]
I would suggest "wl" ,  as CLI contains "l" and so it can be identified as a wrappid CLI tool.

**Reply by anantakumarghosh** [reply]
any more suggestions? @wrappid/core-team

**Reply by saifalamkhan373** [reply]
"wr" could be also good, as the characters on keyboard is near and it will be identified easily for wrappid.

**Reply by samhere17** [reply]
I like "wr". 👍🏽 for @saifalamkhan373

**Reply by techoneel** [reply]
Please visit https://learn.microsoft.com/en-us/windows/terminal/command-line-arguments?tabs=windows

**Reply by anantakumarghosh** [reply]
working as expected on windows 10, linux

**Reply by techoneel** [reply]
Well not working in windows 11

**Reply by anantakumarghosh** [reply]
Hi, we checked this on windows 11, this is working as expected.

![wt](https://github.com/wrappid/.github/assets/61864488/74de8f6b-59d2-42c9-bbc9-b548f4f998a4)

If you are using development edition of toolkit, try these steps:
- `npm i` inside root dir(toolkit) and toolkit/package
- then inside package, run, `npm i -g`

If you are facing problems, consider reaching out to our [paid support](mailto:antaghosh@gmail.com) : antaghosh@gmail.com

**Reply by anantakumarghosh** [reply]
If you are still facing issues, we recommend you to switch to Debian based distribution like ubuntu. We are trying to support most platforms, currently wrappid toolkit works best on linux.

**Reply by anantakumarghosh** [reply, marked as answer]
Hi @techoneel 

We changed our short initial
Our current set of initials

wrappid:
```terminal
wr
wcli
```
wrappid-dev:
```terminal
wd
wdev
```

**Comment by techoneel** [comment]
Closing this discussion since it is done by @anantakumarghosh 💯
